### PR TITLE
Add a patch to make certificates directory accessable in snappy world.

### PR DESCRIPTION
**Pull request by adglkh**
For the sake of easy setup for secure registry server, the manual steps for signed-certificate generation are not required. Snap is in charge of certificate generation right after the first installation.
There're two available configuration options can be set-able via a config hook. One is registry-domain-name, another is certificate-folder. Please take a look at docker.help for further details.

### CODE REVIEW

**Comment by adglkh** [comment]
Here is the instruction that people can follow to setup secure private registry server, test and verify it from each docker client.
http://paste.ubuntu.com/24736911/

File: bin/help [hunk 1]
```diff
@@ -12,12 +13,33 @@ please run the following command to add the login user into docker group.
...
+you need to connect the home interface as it's not auto-connected by default.
     sudo snap connect docker:home :home
 
-Secondly, reload the snap to take effect.
-    snap disable docker
-    snap enable  docker
-
 Then have fun with docker in snappy.
+
+To setup a secure private registry, a typical command to launch server as following.
```
Comment (psftw) [inline bin/help]: Can we take this out?  It's better left in the README, which we could refer to here?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
+# Docker-snap
```
Comment (psftw) [inline README.md]: I want to make a bunch of changes to the README, but will propose them after this initial version gets merged.

File: bin/help (hunk 1 above)
Comment (adglkh) [inline bin/help]: Right. 
It's little bit overkilled that a private registry setup is mentioned twice in our repo somewhere. I'll fix it now.

**Pull request merged**